docs/topics/pages.rst

```diff
@@ -324,6 +324,20 @@ The template can be changed on a per-instance basis by defining a ``get_template
 In this example, pages that have the ``use_other_template`` boolean field set will use the ``blog/other_blog_page.html`` template. All other pages will use the default ``blog/blog_page.html``.
 
 
+Ajax Templates
+~~~~~~~~~~~~~~
+
+If you want to AJAX a page and have different output, other than your default template, you can set an ``ajax_template``
```

Comment:
Not sure if it's clear what "AJAX a page" means here. My suggested wording:

If you want to add AJAX functionality to a page, such as a paginated listing that updates in-place on the page rather than triggering a full page reload, you can set the ``ajax_template`` attribute to specify an alternative template to be used when the page is requested via an AJAX call (as indicated by the `X-Requested-With: XMLHttpRequest` HTTP header):